planetiler-core/src/main/java/com/onthegomap/planetiler/mbtiles/Mbtiles.java

```diff
@@ -181,15 +168,14 @@ private Mbtiles execute(String... queries) {
     return execute(Arrays.asList(queries));
   }
 
-  public Mbtiles addTileIndex() {
-    if (compactDb) {
-      return execute(ADD_TILES_SHALLOW_INDEX_SQL);
-    } else {
-      return execute(ADD_TILE_INDEX_SQL);
-    }
-  }
-
-  public Mbtiles createTables() {
+  /**
+   * Creates the required tables (and views).
+   *
+   * @param skipIndexCreation skip index creation on some tables - in that case the indexes should be added later
+   *                          manually
+   * @return {@code this}
+   */
+  public Mbtiles createTables(boolean skipIndexCreation) {
```

Comment:
it's a little hard to follow what the callers want with the true/false flag... could we make add a `createTablesWithoutIndex` so it's a bit more clear?